compiler/src/dmd/attrib.d

```diff
@@ -845,6 +845,12 @@ extern (C++) final class UnpackDeclaration : AttribDeclaration
             decl = null;
             lowered = true;
         }
+        static import dmd.errors;
+        if (auto uda = userAttribDecl)
+        {
+            dmd.errors.error(loc, "user defined attributes are not supported yet on unpack declarations");
+            return fail();
+        }
```

Comment:
BTW the reason this is in `lower` is because the semantic `visit` call in dsymbolsem.d doesn't seem to get called.